sql/flow_config_test.go

```diff
@@ -26,10 +26,19 @@ func TestGetPypiVersionInvalidHTTPRequestFailure(t *testing.T) {
 	assert.ErrorContains(t, err, expectedErrContains)
 }
 
-func TestGetPypiVersionInvalidAstroCliVersionFailure(t *testing.T) {
-	_, err := GetPypiVersion("https://raw.githubusercontent.com/astronomer/astro-sdk/astro-cli/sql-cli/config/astro-cli.json", "foo")
-	expectedErrContains := "error parsing response for SQL CLI version"
-	assert.ErrorContains(t, err, expectedErrContains)
+func TestGetPypiVersionInvalidAstroCliVersionExactMatchSuccess(t *testing.T) {
+	_, err := GetPypiVersion("https://raw.githubusercontent.com/astronomer/astro-sdk/1673-minor-version-match/sql-cli/config/astro-cli.json", "1.9.0")
```

Comment:
```suggestion
	_, err := GetPypiVersion("https://raw.githubusercontent.com/astronomer/astro-sdk/astro-cli/sql-cli/config/astro-cli.json", "1.9.0")
```
Needs to be corrected to this after [PR](https://github.com/astronomer/astro-sdk/pull/1706) is merged